src/gc-heap-snapshot.cpp

```diff
@@ -446,7 +453,6 @@ static inline void _record_gc_edge(const char *node_type, const char *edge_type,
     auto to_node_idx = record_node_to_gc_snapshot(b);
 
     auto &from_node = g_snapshot->nodes[from_node_idx];
-    from_node.type = g_snapshot->node_types.find_or_create_string_id(node_type);
```

Comment:
🤔 what was in the node_type param here?

Reply:
It got passed into the function. It's an argument to it, I still need to remove it :)